evcouplings/couplings/mapping.py

```diff
@@ -307,6 +306,41 @@ def to_model(self, x):
         return self.__map(x, self.target_to_model)
 
 
+def segment_map_positions_single_column(dataframe, mapper, column):
```

Comment:
Hm - somehow to me this does not quite warrant being a separate function

I think this would be a cleaner solution (and also support your use case of mapping only a single column):
```python
def segment_map_ecs(ecs, mapper, map_i=True, map_j=True)
```

and then

```python
if map_i:
    _map_column("i")

if map_j:
    _map_column("j")
```